### PR TITLE
Adding SAP HANA on SLE 15 Lab

### PR DESCRIPTION
**Pull request by ZeTopHat**

### CODE REVIEW

File: vagrantfiles/hana15/Vagrantfile [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+      s.args = "SLE15"
+      s.env = {
+        "SLEREGCODE" => secret_vars["sleregcode"],
+        "IPADDRESS" => "#{custom_vars["network"]["primarysubnet"]}#{machine_vars["ipaddress"]}",
+        "FQDN" => machine_vars["fqdn"],
+        "SHORT" => machine_vars["short"],
+        "NTPSERVER" => machine_vars["ntpserver"]
+      }
+    end
+    sle.vm.synced_folder "#{WD}provisioners/lab/#{LABNAME}/hanaiso/", "/opt/"
```
Comment (Thr3d) [inline vagrantfiles/hana15/Vagrantfile:82]: nfs-client package must be installed before this

File: vagrantfiles/hana15/Vagrantfile [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
```
Comment (Thr3d) [inline vagrantfiles/hana15/Vagrantfile:123]: nfs-client package must be installed before this

**Review by Thr3d — CHANGES_REQUESTED** [review]
Register SLES and install nfs-client package before using `sle.vm.synced_folder` or deployment fails

File: vagrantfiles/hana15/Vagrantfile (hunk 2 above)
Comment (ZeTopHat) [reply vagrantfiles/hana15/Vagrantfile]: hardcoded rsync as method since your machine is defaulting different than mine.

**Review by Thr3d — DISMISSED** [review]

**Review by Thr3d — CHANGES_REQUESTED** [review]
Because the database is already running on hana15n1 during bootstrap, when the cluster starts and probes rsc_SAPHana_HXE_HDB00 on hana15n1 the monitor fails expecting the db to be stopped.
Perhaps just wait for it to settle and cleanup the expected failure at the end of the bootstrap?
```
crm_resource --wait
crm resource cleanup
```

**Pull request merged**